### PR TITLE
PR for explore page

### PR DESCRIPTION
**Pull request by narharikale**
- added explore page
- added categories filter

Note : Not Responsive Yet

### CODE REVIEW

File: src/frontend/context/videos-context.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    useEffect( () => {
+        (async function getCategories(){
+            try{
+                const { data } = await axios.get('/api/categories')
+                setCategories(data.categories)
+            }catch(error){
+                console.error(error)
+            }
+        })()
+    } , [])
```
Comment (horsemaker) [inline src/frontend/context/videos-context.js]: This can be done in a single useEffect

File: src/frontend/screen/index.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
-export { Signin } from './Auth/Signin'; 
-export { Signup } from './Auth/Signup';
-export { Explore } from './Explore/Explore';
-export { NotFound } from './NotFound/NotFound'
\ No newline at end of file
+
+
+
+
+
```
Comment (horsemaker) [inline src/frontend/screen/index.js]: Empty Space?

**Review by horsemaker — APPROVED** [review]
Looks good! 👌

File: src/frontend/component/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import "./Sidebar.css";
+
+function Sidebar() {
+  return (
+    <aside className="sidebar-container">
+      <ul className="sidebar-list">
+        <li className="sidebar-list-item" >
+            <span className="material-icons-outlined">home</span> 
+            Home
+        </li>
```
Comment (t007rushi) [inline src/frontend/component/Sidebar/Sidebar.jsx:10]: can make a component and names will be arr to map here

File: src/frontend/context/videos-context.js (hunk 1 above)
Comment (t007rushi) [reply src/frontend/context/videos-context.js]: rightly pointed out by @horsemaker

File: src/index.js [hunk 4]
```diff
@@ -14,10 +13,12 @@ root.render(
...
           <VideoProvider>
-            <App />
+            <FilterProvider>
+                <App />
+            </FilterProvider>
           </VideoProvider>
         </AuthProvider>
       </ThemeProvider>
-    </BrowserRouter>  
+    </BrowserRouter>
```
Comment (t007rushi) [inline src/index.js:22]: can make a ```combinedProvider```

**Review by t007rushi — APPROVED** [review]
Great going 
Keep it naru

File: src/frontend/context/videos-context.js (hunk 1 above)
Comment (narharikale) [reply src/frontend/context/videos-context.js]: yes! done

File: src/index.js (hunk 4 above)
Comment (narharikale) [reply src/index.js:22]: i'll implement , thanks

**Pull request merged**